README.md

```diff
@@ -54,21 +54,22 @@ sudo chown -R lorisadmin:lorisadmin /opt/$projectname
 cd /opt/$projectname/bin
 ```
 
-Get the code: Download the latest release from the 
-[releases page](https://github.com/aces/Loris-MRI/releases) 
+Get the code: Download the latest release from the
+[releases page](https://github.com/aces/Loris-MRI/releases)
 and extract it to `/opt/$projectname/bin/mri`
 
 #### 2. Install Python 3 with `pip` and `virtualenv`
 
 ```bash
-sudo apt install python3 
-sudo apt install python3-dev
+sudo apt install software-properties-common
+sudo add-apt-repository ppa:deadsnakes/ppa
+sudo apt update
+sudo apt install python3.12 python3.12-dev python3.12-distutils python3.12-venv
```

Comment:
Some Python versions had problems with `pip` in the virtual environment without `python3.x-distutils`.